editor/resources/templates/template.camera

```diff
@@ -1,5 +1,5 @@
 aspect_ratio: 1.0
-fov: 45.0
+fov: 0.785
```

Comment:
We had a discussion in the team and realised we might want to present this value in degrees later on (or toggle degrees/radians). For that purpose 0.785**4** will round more nicely to 45 degrees.